posts/Fabio/2018-02-13-hibernate-ogm-5-3-CR1-released.adoc

```diff
@@ -0,0 +1,58 @@
+= "Hibernate OGM 5.3 CR1 is out"
+Fabio Massimo Ercoli
+:awestruct-tags: [ "Hibernate OGM", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+We're happy to announce new release:
+http://hibernate.org/ogm/releases/5.3/#get-it[Hibernate OGM 5.3 CR1]
+
+All the changes are described in the
+https://hibernate.atlassian.net/secure/ReleaseNote.jspa?projectId=10160&version=31626
+
+== Versions Upgrade
+
+* Hibernate ORM version is upgraded to 5.2
+* Hibernate Search version is upgraded to 5.9
+* Hibernate Query Parser is upgraded to 1.5
+
+== Infinispan embedded sequences and id generation improvements
+
+Infinspan Clustered Counters for sequences generation with Infinispan embedded are now created at startup.
+Creation of all not declared (created by OGM) counters for sequence generation is created now at startup,
+avoiding in this way the need to check if the sequence exists at runtime.
+
+Infinspan Clustered Counter names for id generation also take consideration the table
+and not only the columname in Infinispan embedded.
+In this way using the `table` attribute we can distinguish counters using the same `pkColumnValue` value
+in `@TableGenerator` annotation.
```

Comment:
I would remove this sentence and maybe give an example of the name generated for a sample entity.